sys-fs/bees/bees-9999.ebuild

```diff
@@ -48,7 +45,14 @@ pkg_pretend() {
 			ewarn "# WARNING: CPU: 3 PID: 18172 at fs/btrfs/backref.c:1391 find_parent_nodes+0xc41/0x14e0"
 			ewarn
 		fi
-
+		if kernel_is -lt 5 0 4; then
+			ewarn "IMPORTANT: With kernel versions below 5.0.4, you may experience data corruption"
+			ewarn "due to bees using compression in btrfs. You are adviced to use a chronologically"
+			ewarn "later kernel, that includes older LTS versions released after 5.0.4:"
+			ewarn "Fixed in: 5.1+, 5.0.4+, 4.19.31+, 4.14.108+, 4.9.165+, 4.4.177+, 3.18.137+"
```

Comment:
The `if` should probably check for 5.1.0 then. This was probably my fault in a previous commit.